### PR TITLE
AOM support never landed, need to keep making install.rdf #324

### PR DESCRIPTION
**Pull request by janodvarko**
This patch worked for me. Can anyone review please?
Honza

### CODE REVIEW

**Comment by jsantell** [comment]
Looks good -- merging this in

**Comment by janodvarko** [comment]
Thanks Jordan!

Honza

File: lib/settings.js [hunk 1]
```diff
@@ -4,13 +4,13 @@
 "use strict";
 
 // Set minimum version to 38.0a1, because some SDK changes are necessary.
-// Must be "38.0a1", as 38.0a1 < 33.0 according to how Firefox does versioning
+// Must be "38.0a1", as 38.0a1 < 38.0 according to how Firefox does versioning
 exports.MIN_VERSION = "38.0a1";
 // Set a MAX_VERSION to use in install.rdf - this needs to be updated every once in a while
 // see this page for supported ranges: https://addons.mozilla.org/en-US/firefox/pages/appversions/
-exports.MAX_VERSION = "39.0";
+exports.MAX_VERSION = "43.0";
```
Comment (benmccann) [inline lib/settings.js]: should this be 44.0? i see it listed in https://addons.mozilla.org/en-US/firefox/pages/appversions/

Comment (jsantell) [inline lib/settings.js]: Not until September 21st, I'd wager

Comment (jsantell) [inline lib/settings.js]: The previous issue is due to `AOM_SUPPORT_VERSION` though

**Pull request merged**